readme.md

```diff
@@ -4,13 +4,13 @@
 [![Docs](https://docs.rs/tokio-bin-process/badge.svg)](https://docs.rs/tokio-bin-process)
 [![dependency status](https://deps.rs/repo/github/shotover/tokio-bin-process/status.svg)](https://deps.rs/repo/github/shotover/tokio-bin-process)
 
-Allows your integration tests to run your application under a separate process and assert on tokio-tracing events.
+Allows your integration tests to run your application under a separate process and assert on `tracing` events.
```

Comment:
hmmm, maybe `tokio-rs/tracing` and/or link to https://github.com/tokio-rs/tracing
Even though the project is called tracing, just `tracing` is ungoogleable and ambiguous.